### ISSUE TITLE
Having 2 apps with similar package both badges are updated

### ISSUE THREAD

**Issue by khrizt** [leolin310148/ShortcutBadger#75]
Hi,

I have 2 published apps with package names:

```
com.[company].[app_name_1]
com.[company].[app_name_2]
```
And when executing

`ShortcutBadger.with(this).count(numUnreadNotifications);`

both apps get the badge of the number I set... is that a bug? Can I change something in configuration?

Thanks

**Comment by lalon** [comment]
Does your launcher class has the same path and file name in both apps?
Check if changing them solves the issue

**Comment by khrizt** [comment]
Yep that probably solves the issue but creates a bigger problem with Manifest mergers and different launcher activities to manage... right now it's not an option. :(

**Comment by Waboodoo** [comment]
I have the same problem. Is there any other way to resolve this?

**Comment by gradha** [comment]
I'm using different flavours which create separate binaries for debug, test and production environments. I was quite shocked to see the debug app changing the badge of all the other variants. Apparently on LG-E430 with Android 4.1 the intent sent through `DefaultBadger` ignores the `INTENT_EXTRA_PACKAGENAME`.

**Comment by yu-t** [comment]
The same is for me on LG g5.
